lib/recurly/apple-pay.js

```diff
@@ -158,6 +169,23 @@ class ApplePay extends Emitter {
     });
   }
 
+  mapPaymentData (inputs, data) {
+    inputs.paymentData = data.token.paymentData;
+    inputs.paymentMethod = data.token.paymentMethod;
+
+    FIELDS.forEach(field => {
+      if (inputs[field] || !BILLING_DATA_MAP[field]) return;
+
+      let tokenData = data.billingContact[BILLING_DATA_MAP[field]];
+
+      // address lines are an array from Apple Pay
+      if (field === 'address1') tokenData = tokenData[0];
+      else if (field === 'address1') tokenData = tokenData[1];
```

Comment:
should this be `'address2'`?

Reply:
fixed this typo and added some basic specs